sktime/performance_metrics/forecasting/_functions.py

```diff
@@ -250,6 +253,9 @@ def mean_absolute_scaled_error(
             "The M4 Competition: 100,000 time series and 61 forecasting methods",
             International Journal of Forecasting, Volume 3
     """
+    y_train = kwargs.pop("y_train", None)
+    if y_train is None:
+        raise ValueError("y_train must be supplied to compute this metric.")
```

Comment:
Perhaps something like this?
```suggestion
        raise ValueError(f"{self.name} requires `y_train`. Please pass `y_train` as a keyword argument when calling the metric")
```

Reply:
I like that. Updated